### ISSUE TITLE
Compile with old GLIBC or musl?

### ISSUE THREAD

**Issue by dufferzafar** [adamritter/fastgron#12]
I'll try compiling on my box as well, but just wanted to leave this here:

I'm on an old Ubuntu 20.04 Focal

```
❯ fastgron
fastgron: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by fastgron)
fastgron: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by fastgron)
```

**Comment by adamritter** [comment]
Musl sounds like a great idea, static compilation is probably more portable than using an older glibc.

**Comment by dufferzafar** [comment]
Yeah, I've seen rust tools shipping with a musl build precisely for this reason.

Of C++ applications, lnav is one that ships with a musl build: https://github.com/tstack/lnav/blob/master/.github/actions/muslbuilder/entrypoint.sh

**Comment by adamritter** [comment]
Cool, if you want to fix the release, you can start with changing the build here and send me a PR:

https://github.com/adamritter/fastgron/blob/main/.github/workflows/build.yml

Still, I  appreciate your feedback :)
